### PR TITLE
Fix React Native Android

### PR DESCRIPTION
**Pull request by muhamad-rizki**
Hi @gutenye 
Thanks for your great library, here I create a PR to fix some issues on android
- removing arm_neon.h
- ~added gradle hook so the models can be embedded automatically~
- fixing convert std vector text to std vector text lines

### CODE REVIEW

File: packages/react-native/example/android/app/build.gradle [hunk 1]
```diff
@@ -179,3 +179,4 @@ dependencies {
 
 apply from: new File(["node", "--print", "require.resolve('@react-native-community/cli-platform-android/package.json', { paths: [require.resolve('react-native/package.json')] })"].execute(null, rootDir).text.trim(), "../native_modules.gradle");
 applyNativeModulesAppBuildGradle(project)
+apply from: file("../../node_modules/@gutenye/ocr-react-native/ocr.gradle")
```
Comment (gutenye) [inline packages/react-native/example/android/app/build.gradle:182]: Thanks for the PR. Don't need this, models are in 
 https://github.com/gutenye/ocr/tree/main/packages/react-native/android/src/main/assets folder, they are copied automatically during the build.

Comment (muhamad-rizki) [reply packages/react-native/example/android/app/build.gradle:182]: I see, I'll remove them later, in the latest package I don't get the models embedded to the apps, that's why I created those

Comment (gutenye) [reply packages/react-native/example/android/app/build.gradle]: Are you on Windows? I used a symlink, maybe the symlink is not working for you?

**Comment by gutenye** [comment]
tACK awesome, finally arm_neon.h is removed

File: packages/react-native/example/android/app/build.gradle (hunk 1 above)
Comment (muhamad-rizki) [reply packages/react-native/example/android/app/build.gradle]: > Are you on Windows? I used a symlink, maybe the symlink is not working for you?

nope, I use Arch linux, after looking it up there seems no asset folder on the android folder, that's why the models are not embedded
image
It's on v1.4.4 btw

Comment (gutenye) [reply packages/react-native/example/android/app/build.gradle]: @muhamad-rizki I published 1.4.6 version based on https://github.com/gutenye/ocr/pull/17, can you test it and leave a message there?

**Pull request merged**